app/src/androidTest/java/org/dharmaseed/android/TalkRepositoryTest.java

```diff
@@ -55,28 +68,13 @@ public void getAllTalks_isCorrect()
     {
         talkRepository = new TalkRepository(dbManager);
 
-        Cursor actualCursor = talkRepository.getTalks(null, null, false, false);
```

Comment:
Here's the change we discussed (requiring callers to specify exactly which columns they want).  There are similar changes in a few other tests below.